Tests/Sitemap/Url/GoogleVideoUrlDecoratorTest.php

```diff
@@ -26,6 +29,95 @@ class GoogleVideoUrlDecoratorTest extends TestCase
     protected $xml;
 
     protected function setUp()
```

Comment:
Can you add a test case for a video url with multiple video object, with some assertions on the generated xml ?